### PR TITLE
OSX Code Signing & Notarization Guide for Tauri Bundler

### PR DESCRIPTION
**Pull request by ImmaZoni**
Created a basic guide on how to sign a .dmg file with Tauri bundler via GitHub actions.

Let me know if you have any modifications or suggestions.

### CODE REVIEW

File: docs/en/usage/guides/bundler/sign-osx.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+sidebar_label: OSX Code-signing with GitHub Actions
+---
+
+import Alert from '@theme/Alert'
+
+## Intro
+
+Tauri has a smooth code-signing & notarization functionality built directly into the bundler and configured via the `tauri.conf.json` 
+
+This guide will give a brief overview of how to sign an application, and how to get the app notarized with apple. All in a GitHub action.
```
Comment (nothingismagick) [inline docs/en/usage/guides/bundler/sign-osx.md]:
```suggestion
This guide will give a brief overview of how to sign an application, and how to get the app notarized with Apple. All in a GitHub action. 
```

File: docs/en/usage/guides/bundler/sign-osx.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+## GitHub Secrets
+
+We will need to add a few GitHub secrets for the proper configuration of the GitHub Action. These can be named however you would like, but we must assign them to the correct Tauri variables, so keep them as relevant as possible. 
+- You can view [this](https://docs.github.com/en/actions/reference/encrypted-secrets) guide for how to add GitHub secrets. 
+
+The secrets I used are as follows
+
+| GitHub Secrets | Value for Variable |
+|     :---:      |        :---:            |
+|APPLE_CERTIFICATE| Base64 encoded version of your .p12 certificate. guide [here](https://localazy.com/blog/how-to-automatically-sign-macos-apps-using-github-actions#reference)|
```
Comment (nothingismagick) [inline docs/en/usage/guides/bundler/sign-osx.md]:
```suggestion
|APPLE_CERTIFICATE| Base64 encoded version of your .p12 certificate. You can find a guide [here](https://localazy.com/blog/how-to-automatically-sign-macos-apps-using-github-actions#reference)|
```

**Review by nothingismagick — APPROVED** [review]
Wow - thankyou so much. This is great!!!

**Comment by lucasfernog** [comment]
Can you PR the same to the tauri repo? That's is where the file should live.

**Pull request merged**